cms/djangoapps/contentstore/views/videos.py

```diff
@@ -468,3 +482,35 @@ def is_status_update_request(request_data):
     Returns True if `request_data` contains status update else False.
     """
     return any('status' in update for update in request_data)
+
+def _get_validated_params(request):
+    """
+    Reads and returns valid parameters
+    """
+    params = { "paginated": True }
+
+    page = request.GET.get("page")
+    page_size = request.GET.get("page_size")
+    search_key = request.GET.get("search_key")
+    sort_field = request.GET.get("sort_field")
+    sort_dir = request.GET.get("sort_dir")
+
+    # Page must be an integer greater than 0
+    try:
+        if int(page) > 0: params["page"] = int(page)
+    except:
+        pass
```

Comment:
Why are we not doing anything when page is zero or less ? and when does page has zero value ?